#return the point cloud of the respective cell
string cellID
---
pr2_pick_perception/ClusterList locations
